test/tests/tracesConverterTest.js

```diff
@@ -463,4 +472,526 @@ describe('Traces converter tests', function () {
+            actor: {
+                name: name,
+                account: {
+                    homePage: 'http://a2:3000/',
+                    name: 'Anonymous'
+                }
+            },
+            verb: {
+                id: 'http://.../asddsfdg'
+            },
+            object: {
+
+            },
+            result: {
+                extensions: {
+                    '23.423.4756/ext4': '434'
+                }
+            },
+            timestamp: timestamp
+        };
+        var result = getRealtimeData(statement);
+        var trace = result.trace;
+        should(result.error.message).eql('Object Id is missing for statement, ' + statement);
+        should(trace).eql(undefined);
+
+        done();
+    });
+
+    it('should not convert a statement with an object id ending with /', function (done) {
+
+        var timestamp = '2016-05-16T11:48:25Z';
+        var name = 'testName';
+        var statement = {
+            id: '19de3bf2-6b7f-4399-a71b-da5f3674c8f8',
+            actor: {
+                name: name,
+                account: {
+                    homePage: 'http://a2:3000/',
+                    name: 'Anonymous'
+                }
+            },
+            verb: {
+                id: 'http://.../asddsfdg'
+            },
+            object: {
+                id: 'http://..objectId/'
+            },
+            result: {
+                extensions: {
+                    '23.423.4756/ext4': '434'
+                }
+            },
+            timestamp: timestamp
+        };
+        var result = getRealtimeData(statement);
+        var trace = result.trace;
+        should(result.error.message).eql('Object Id cannot end with / for statement, ' + statement);
+        should(trace).eql(undefined);
+
+        done();
+    });
+
+
+    it('should not convert a statement with a missing object definition', function (done) {
+
+        var timestamp = '2016-05-16T11:48:25Z';
+        var name = 'testName';
+        var statement = {
+            id: '19de3bf2-6b7f-4399-a71b-da5f3674c8f8',
+            actor: {
+                name: name,
+                account: {
+                    homePage: 'http://a2:3000/',
+                    name: 'Anonymous'
+                }
+            },
+            verb: {
+                id: 'http://.../asddsfdg'
+            },
+            object: {
+                id: 'http://../objectId'
+            },
+            result: {
+                extensions: {
+                    '23.423.4756/ext4': '434'
+                }
+            },
+            timestamp: timestamp
+        };
+        var result = getRealtimeData(statement);
+        var trace = result.trace;
+        should(result.error.message).eql('Object definition is missing for statement, ' + statement);
+        should(trace).eql(undefined);
+
+        done();
+    });
+
+    it('should not convert a statement with a missing object definition type', function (done) {
```

Comment:
Any previous statement has not an object definition but not throw an error. If someone read the previous tests but not this, He can think that definition is not necessary.